fiftyone/utils/labelbox.py

```diff
@@ -620,10 +620,13 @@ def upload_samples(self, samples, anno_key, backend):
         project_name = config.project_name
         members = config.members
         classes_as_attrs = config.classes_as_attrs
-        is_video = (samples.media_type == fomm.VIDEO) or (
-            samples.media_type == fomm.GROUP
-            and samples.group_media_types[samples.group_slice] == fomm.VIDEO
-        )
+        is_video = samples.media_type == fomm.VIDEO
+
+        if samples._is_clips:
+            raise ValueError(
+                "The Labelbox backend does not yet support annotating clip "
+                "views"
+            )
```

Comment:
What do you think about moving this into `annotations.py` and adding a `supports_clip_views` property to each annotation backend that defaults to `False` (With CVAT returning `True`). I'm just thinking about the darwin and other non builtin integrations which would be good to prevent from needing to be manually updated to prevent annotation of clip views.

Reply:
Great idea! Just added ✅